orttraining/orttraining/python/ort_trainer.py

```diff
@@ -415,6 +430,12 @@ def create_ort_training_session_with_optimizer(model, device, training_optimizer
     torch_params = {}
     optimizer_attributes_map = {}
     optimizer_int_attributes_map = {}
+
+    unused_frozen_weights = [n for n in frozen_weights if n not in [i.name for i in model.graph.initializer]]
+    if unused_frozen_weights:
+        warnings.warn("Ignoring {} in frozen_weights as they are not found in model weights."\
```

Comment:
Should this be just a warning or an assert error? Wrong parameter names is usually a serious mistake that should be fixed